singleplayer/scenarios/scenario022.lua

```diff
@@ -1,652 +1,652 @@
-			RgbColor = 0.99609375 0.546875 0;
-			AllyTeam = 0;
-			TeamLeader = 0;
-			StartPosX = 4888;
-			StartPosZ = 3712;
-		}
-	
-		[allyTeam2]
-		{
-			numallies = 0;
-		}
-	
-		[allyTeam1]
-		{
-			numallies = 0;
-		}
-	
-		[team3]
-		{
-			Side = Cortex;
-			Handicap = __ENEMYHANDICAP__;
-			RgbColor = 0.99609375 0.546875 0;
-			AllyTeam = 2;
-			TeamLeader = 0;
-			StartPosX = 846;
-			StartPosZ = 4430;
-		}
-	
-		[team0]
-		{
-			Side = __PLAYERSIDE__;
-			Handicap = __PLAYERHANDICAP__;
-			RgbColor = 0.99609375 0.546875 0;
-			AllyTeam = 0;
-			TeamLeader = 0;
-			StartPosX = 1176;
-			StartPosZ = 1414;
-		}
-	
-		[team2]
-		{
-			Side = Cortex;
-			Handicap = __ENEMYHANDICAP__;
-			RgbColor = 0.99609375 0.546875 0;
-			AllyTeam = 2;
-			TeamLeader = 0;
-			StartPosX = 5164;
-			StartPosZ = 615;
-		}
-	
-		[modoptions]
-		{
-			deathmode = builders;
-			maxunits = 2000;
-			map_waterlevel = 150;
-			startenergy = 1500;
-			startmetal = 500;
-			scenariooptions = __SCENARIOOPTIONS__;
-		}
-	
-		[ai2]
-		{
-			Host = 0;
-			IsFromDemo = 0;
-			Name = BARbstable(3);
-			ShortName = BARb;
-			Team = 3;
-			Version = stable;
-		}
-	
-		[ai0]
-		{
-			Host = 0;
-			IsFromDemo = 0;
-			Name = BARbstable(1);
-			ShortName = BARb;
-			Team = 1;
-			Version = stable;
-		}
-	
-		[player0]
-		{
-			IsFromDemo = 0;
-			Name = __PLAYERNAME__;
-			Team = 0;
-			rank = 0;
-		}
-	
-		NumRestrictions=__NUMRESTRICTIONS__;
-	
-		[RESTRICT]
-		{
-			__RESTRICTEDUNITS__
-		}
-	
-		hostip = 127.0.0.1;
-		hostport = 0;
-		numplayers = 1;
-		startpostype = 3;  // 0 fixed, 1 random, 2 choose in game, 3 choose before game (see StartPosX)
-		mapname = __MAPNAME__;
-		ishost = 1;
-		numusers = 4;
-		gametype = __BARVERSION__;
-		GameStartDelay = 7; // seconds before game starts after loading/placement
-		myplayername = __PLAYERNAME__;
-		nohelperais = 0;
-	}
-	]],
-
-}
-
-return scenariodata
+local scenariodata = {
+	index			= 22, --  integer, sort order, MUST BE EQUAL TO FILENAME NUMBER
+	scenarioid		= "Begin06", -- no spaces, lowercase, this will be used to save the score and can be used gadget side
+    version         = "1", -- increment this to reset the score when changing a mission, as scores are keyed by (scenarioid,version,difficulty)
+	title			= "Supremacy", -- can be anything
+	author			= "wilkubyk", -- your name here
+	isnew = true,
+	imagepath		= "scenario022.jpg", -- placed next to lua file, should be 3:1 ratio banner style
+	imageflavor		= "River Assault", -- This text will be drawn over image
+    summary         = [[After a ferocious battle, a Cortex Commander decided to join Armada forces. Unfortunately, other Cortex commanders discovered the intent of the treacherous plan and sent a mighty force to prevent it.]],
+	briefing 		= [[You will start with some Lazarus (Resurrection and Repair Bots), that can resurrect units from their wrecks. Your Only Hope is to hide your position and build a mighty army to crush all opposing forces.

```

Comment:
only still uppercase?

Reply:
I thought it was a star wars reference but didn't want to make it too obvious adding the My